code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -38,7 +38,7 @@
 	. = (loc == user && istype(over, /obj/screen)) || ..()
 
 /obj/item/storage/handle_mouse_drop(var/atom/over, var/mob/user)
-	if(canremove && (ishuman(user) || isrobot(user)))
+	if(canremove && (ishuman(user) || isrobot(user) || isanimal(user)) && !user.incapacitated(INCAPACITATION_DISRUPTED))
```

Comment:
Probably needs `()` around that `&&`. What is this meant to solve?

Reply:
I added animals there because passing living will include AI and other mobs I didn't expect.

Reply:
Actually the idea I wanted is allow Nymphs and mouses to look inside bags.